mimic/test/test_fastly.py

```diff
@@ -170,6 +170,116 @@ def test_create_backend(self):
         self.assertEqual(200, response.code)
         self.assertEqual(sorted(json_body), sorted(domain_details))
 
+    def test_create_condition(self):
+        """
+        ``POST /service/{service_id}/version/{version_id}/condition' against
```

Comment:
Proper ReST formatting is that ``` ``stuff`` ``` is monospaced; ``` ``stuff' ``` would be a syntax error.  Sorry we don't have build automation to check this automatically, we should add some!